ui/src/index.css

```diff
@@ -11,3 +11,8 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.radio-player .audio-main, 
```

Comment:
Is there a way to move this to scoped styles (with `makeStyles`)? I don't like adding global styles to the app like this, unless strictly necessary